CIScripts/Build/BuildFunctions.ps1

```diff
@@ -294,6 +288,10 @@ function Invoke-ProductUnitTests {
             "timeout's threshold. They were copied from Linux unit test job.")]
         $Env:TASK_UTIL_RETRY_COUNT = 6000
 
+        Invoke-NativeCommand -ScriptBlock {
+            scons --opt=$BuildMode kernel-tests vrouter:test
+        }
```

Comment:
Why not Out-Null'ed?

Reply:
1. Because cut-pasted from the old code.
2. Because I don't care since it'll be removed in #557